daemon/loop.go

```diff
@@ -149,6 +150,52 @@ func (d *Daemon) pullAndSync(logger log.Logger) {
 		}); err != nil {
 			logger.Log("err", err)
 		}
+
+		// Find notes in revisions.
+		for i, rev := range revisions {
+			// Skip last revision (since we requested HEAD~1 above)
+			if i == len(revisions)-1 {
```

Comment:
Are you saying that because it asks for `synctag~1..HEAD`, it will get _at least_ `HEAD~1..HEAD`? If so, and the code can't be more explicit about what's happening, I think it's worth having an explanation at the line mentioned -- i.e., why do we even ask for `synctag~1..HEAD` in the first place?